values-schema.yaml

```diff
@@ -1959,6 +1959,24 @@ properties:
             x-secret: 'randAlphaNum 20'
         required:
           - postgresqlPassword
+      kubeclarity:
+        description: Kubeclarity security scanning Helm chart.
+        properties:
+          _rawValues:
+            $ref: '#/definitions/rawValues'
+          enabled:
+            type: boolean
+            default: false
+          logLevel:
+            $ref: '#/definitions/logLevel'
```

Comment:
There is a mismatch between enums from `#/definitions/logLevel` ( trace, debug, info, warn , error ) and enums defined in kubeclarity chart (debug, info, warning, error, fatal, panic).

Since they cannot be mapped one to one, then I propose to define inline enum that contains kubeclarity constants.
If you do, so then please do not forget to update enum in spec at otomi-api.

Reply:
Wow great catch!